src/puppetlabs/dujour/version_check.clj

```diff
@@ -0,0 +1,116 @@
+(ns puppetlabs.dujour.version-check
+  (:require [clojure.tools.logging :as log]
+            [schema.core :as schema]
+            [ring.util.codec :as ring-codec]
+            [puppetlabs.http.client.sync :as client]
+            [cheshire.core :as json]
+            [trptcolin.versioneer.core :as version]
+            [slingshot.slingshot :as sling]))
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;; Constants
+
+(def default-group-id "puppetlabs.packages")
+(def default-update-server-url "http://updates.puppetlabs.com")
```

Comment:
I basically ported all of this straight over from the dujour version check service in Puppet Server, which doesn't support polling a server with SSL configured, which I'm guessing is why the URL is http rather than https.